src/main/java/ru/clevertec/userservice/user/model/request/CreateJournalistRequest.java

```diff
@@ -0,0 +1,4 @@
+package ru.clevertec.userservice.user.model.request;
+
+public class CreateJournalistRequest extends CreateUserRequest {
```

Comment:
remove this empty class, if CreateJournalistRequest add no logic or additional fields than use just CreateUserRequest object